oap-server/server-starter/src/main/resources/meter-analyzer-config/python-runtime.yaml

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+expSuffix: instance(['service'], ['instance'], Layer.GENERAL)
+metricPrefix: meter
+metricsRules:
+  - name: instance_pvm_process_cpu_utilization
+    exp: instance_pvm_process_cpu_utilization
+  - name: instance_pvm_process_mem_utilization
+    exp: instance_pvm_process_mem_utilization
+  - name: instance_pvm_total_cpu_utilization
+    exp: instance_pvm_total_cpu_utilization
+  - name: instance_pvm_total_mem_utilization
+    exp: instance_pvm_total_mem_utilization
+  - name: instance_pvm_gc_g0
+    exp: instance_pvm_gc_g0
+  - name: instance_pvm_gc_g1
+    exp: instance_pvm_gc_g1
+  - name: instance_pvm_gc_g2
+    exp: instance_pvm_gc_g2
```

Comment:
Are these GC counters? It feels increasing always like so.

Reply:
> Nope. It's the number of objects for each generation.
Like Java, the objects from older generations are more infrequently to be checked if it can be deleted.

Oh, I don't ask about what `g1` and `g2` mean. I mean is the number a counter or a gauge? What does it mean in every report period? Do we need increase for them too?

Reply:
Gauge.
It's the number of objects that each generation keeps at the moment of reporting. We don't need to increase.

Reply:
Then it reports every 20s, but it is downsampling to minute.
So, we are calculating avg object number in the duration.

Reply:
> Then it reports every 20s, but it is downsampling to minute. So, we are calculating avg object number in the duration.

Yes. Is this okay?

Reply:
@arugal Could you confirm this is from design? 
There are two options, one is using the current way, the other is using the Max as downsampling policy, which shows the worst case.

Reply:
Sorry, should ping @Superskyyy @kezhenxu94 Who is your mentor?

Reply:
> Sorry, should ping @Superskyyy @kezhenxu94 Who is your mentor?

@kezhenxu94

Reply:
Yes, this is reasonable to me.